pom.xml

```diff
@@ -105,7 +105,7 @@
             <dependency>
                 <groupId>io.netty</groupId>
                 <artifactId>netty-tcnative-boringssl-static</artifactId>
-                <version>2.0.54.Final</version>
+                <version>2.0.60.Final</version>
```

Comment:
I hate to be That Guy, but I think there's a reference in the README that needs to get updated, too.